tests/tensor_search/test_api.py

```diff
@@ -62,6 +63,54 @@ def test_memory_disabled_403(self):
             response = self.client.get("/memory")
             self.assertEqual(response.status_code, 403)
 
+    def test_custom_search_limit(self):
+        """
+        Test that the search endpoint returns the expected search limit when MARQO_MAX_SEARCH_LIMIT is set.
+        """
+        custom_limits = [2000, 1000000]
+        for custom_limit in custom_limits:
+            with patch.dict('os.environ', {
```

Comment:
What if these two env vars are not set? I think we still want to have a default max value, right? which is 1000 and 10000 respectively. And if user want larger Max limit, they can provide it by setting the env var.

Reply:
Hey @papa99do, I think these default values are already set in config.py in api, right?

Reply:
Or were you referring to making a test without setting these env vars explicitly?

Reply:
Got it. Looks good.